auth/AuthSample/Program.cs

```diff
@@ -31,7 +31,7 @@ namespace GoogleCloudSamples
     class BaseOptions
     {
         string _projectId;
-        [Option('p', Default = null, HelpText = "Your Google project id.")]
+        [Option('p', HelpText = "Your Google project id.")]

```

Comment:
All of the other options in this file have mostly null default values. This is mostly right since we need the projectId to be provided by the caller. Setting this default at null explicitly is mostly to make clear to the caller that they need to specify their own projectId.